tensorrt_llm/llmapi/llm_args.py

```diff
@@ -59,6 +59,14 @@
 # TODO[chunweiy]: move the following symbols back to utils scope, and remove the following import
 
 
+@dataclass
+class _CudaGraphConfig:
```

Comment:
For official data classes, we prefer to make them Pydantic, we can make CudaGraphConfig as below:
```python
class CudaGraphConfig(BaseModel):
    cuda_graph_batch_sizes: List[int] = Field(default_factory=list, description="...")
    ...
```